src/hooks/index.ts

```diff
@@ -0,0 +1 @@
+export * from './usePromiseDialog';
```

Comment:
may be just `useDialog`? More clear and users can understand what works with promise using typescript definitions